### DISCUSSION TITLE
Custom CSS file ?

### DISCUSSION THREAD

**Discussion by SheepDomination** [zen-browser/desktop#1923] [Q&A]
How do I add a custom css file ?

**Comment by Axenide** [comment]
Hi, you can do just like in Firefox. Just add your `userChrome.css` to your Zen Browser profile's chrome folder. :)
It has some CSS files but your `userChrome.css` will have priority over them

In my case my profile is in a folder called "Zen", so my `userChrome.css` is in `~/.zen/Zen/chrome/userChrome.css`.

**Reply by SheepDomination** [reply]
In other words; `C:\Users\%username%\AppData\Roaming\zen\Profiles\lz2tcx7f.Default Profile`

**Reply by Axenide** [reply]
Yeah, sorry, I don't use Windows heh.

**Reply by SheepDomination** [reply]
Hopefully someone who uses windows can give some feedback ?

**Comment by aelew** [comment]
1. Go to `about:support` in your browser.
2. Click on `Open Folder` next to `Profile Folder` under `Application Basics`.
3. Your profile folder should open up in your file explorer. Create a `chrome` folder there if there isn't already one.
4. Create a file named `userChrome.css` in your `chrome` folder with your custom CSS.

You may need to restart your browser to see your changes.

**Comment by anatasiajp** [comment]
https://docs.zen-browser.app/guides/live-editing
